stdlib/public/runtime/HeapObject.cpp

```diff
@@ -72,6 +72,19 @@ static inline bool isValidPointerForNativeRetain(const void *p) {
 #endif
 }
 
+// Call the appropriate implementation of the `name` function, passing `args`
+// to the call. This checks for an override in the function pointer. If an
+// override is present, it calls that override. Otherwise it directly calls
+// the default implementation. This allows the compiler to inline the default
+// implementation and avoid the penformance penalty of indirecting through
```

Comment:
Typo: "penformance"

Reply:
Argh. You're not even the first one to point this out. I'm just bad at fixing it.